Cauldron/Assets/Scripts/VisitorsProvider.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections.Generic;
+using CauldronCodebase;
+using UnityEngine;
+
+public class VisitorsProvider
+{
+    private static readonly int CHARACTER_COUNT = 54;
```

Comment:
So, this number is changing literally even when we speak) 

I suggest to use our SODictionary monster to provide it. It knows exactle how many Villagers it has and can Serialize their number and provide it each time LoadInternal() is triggered in SODictionaryLoader

You can check type with if (obj is Villager) count++